<h1> How a drug for COVID-19 is designed?</h1>

<ul>
  <li><a href="#introduction">Introduction</a></li>
  <li><a href="#briefing-on-the-mechanism-of-action-of-sars-cov-2">Briefing on the mechanism of action of SARS-CoV-2</a></li>
  <li><a href="#assignment-questions">Assignment questions</a>
    <ul>
      <li><a href="#1-virus-cell-interactions">1) Virus-cell interactions</a></li>
      <li><a href="#2-variability-in-the-sars-cov-2-genome">2) Variability in the SARS-CoV-2 genome</a></li>
      <li><a href="#3-rational-drug-discovery">3) Rational Drug Discovery</a>
        <ul>
          <li><a href="#ligand">Ligand</a></li>
          <li><a href="#complex">Complex</a></li>
        </ul>
      </li>
    </ul>
  </li>
  <li><a href="#references">References</a></li>
</ul>

<h1 id="introduction">Introduction</h1>

<p>In this assignment for the LVF601M course on Systems Biology at the University of Iceland, we will get some hints on the rational design of a new drug. In particular, we will visualize some simple details on the way an antiviral for SARS-CoV-2 can be designed.</p>

<p>Needed software and databases:</p>

<ol>
  <li><a href="http://openbabel.org/wiki/Main_Page">OpenBabel</a> to convert small molecules coordinates from one format to another. You can also use its <a href="http://www.cheminfo.org/Chemistry/Cheminformatics/FormatConverter/index.html">online version</a>.</li>
  <li><a href="https://www.cgl.ucsf.edu/chimera/">Chimera</a> to visualize proteins and their interactions.</li>
  <li><a href="https://www.uniprot.org/">UNIPROT</a> to obtain information about a particular protein (sequence, structure, interactions…).</li>
  <li><a href="https://www.rcsb.org/">PDB</a> to obtain 3D structures of proteins.</li>
</ol>

<h1 id="briefing-on-the-mechanism-of-action-of-sars-cov-2">Briefing on the mechanism of action of SARS-CoV-2</h1>

<p>The SARS-CoV-2 virus is the cause of the disease known as COVID19. The Spike protein is responsible for anchoring the virus to the cell surface.</p>

<table>
  <thead>
    <tr>
      <th style="text-align: center"><img src="../figures/glyco.png" alt="" /></th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: center">Detail of the structure of the SARS-CoV-2, showing the Spike proteins in their glycosilated form</td>
    </tr>
  </tbody>
</table>

<p>From here the fusion of the membranes occurs and the virus pours its RNA content into the cell. This RNA uses the cellular machinery to replicate the virus and to generate many more that can infect other cells.</p>

<table>
  <thead>
    <tr>
      <th style="text-align: center"><img src="../figures/sars-cov-2-fusion_small.png" alt="" /></th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: center"><a href="https://pdb101.rcsb.org/sci-art/goodsell-gallery/sars-cov-2-fusion">Credit: David S. Goodsell</a></td>
    </tr>
  </tbody>
</table>

<p>A nice introductory video on the wat SARS-CoV-2 infects a cell and on the development of vaccines against it:</p>

<p><a href="https://youtu.be/5DGwOJXSxqg"><img src="https://img.youtube.com/vi/5DGwOJXSxqg/0.jpg" alt="IMAGE ALT TEXT HERE" /></a></p>

<p>The virus machinery includes the information to translate a collection of proteins that, once assembled into new viral particles to infect new host cells. Some of these proteins are sructural and other have a precise funcion to help the maturation of the virus within the cell. See the details of the SARS-CoC-2 proteome in the figure.</p>

<table>
  <thead>
    <tr>
      <th style="text-align: center"><img src="./figures/genome-illustr-2021-update2.png" alt="" /></th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: center">The proteome of SARS-CoV-2</td>
    </tr>
  </tbody>
</table>

<h1 id="assignment-questions">Assignment questions</h1>

<h2 id="1-virus-cell-interactions">1) Virus-cell interactions</h2>

<ol>
  <li>Get the human SARS-CoV-2 virus Spike protein Uniprot code.</li>
  <li>Identify the name of the cell surface protein that the SARS-CoV-2 Spike protein interacts with.</li>
  <li>Look in the PDB database for a structure of the complex between Spike protein and the above cell membrane protein. Add an image of the PDB obtained with Chimera.</li>
  <li>Look in the PDB database for a structure of the complex between Spike protein and antibodies. Add an image of the PDB obtained with Chimera.</li>
  <li>Identify the residues that are in the interface regions, using the <code class="language-plaintext highlighter-rouge">select zone</code> tool in Chimera. Are they many? What do you think a good strategy for preventing SARS-CoV-2 to interact with the cell could be? Are the regions of interaction the same in the complexes you located in the above steps?</li>
</ol>

<h2 id="2-variability-in-the-sars-cov-2-genome">2) Variability in the SARS-CoV-2 genome</h2>

<p>Go to the SARS-CoV-2 genome variation site at Stanford University: <a href="https://covdb.stanford.edu/variants/omicron_ba_1_3/">COVDB</a>. Look for the page devoted to the omicron variants:</p>

<ol>
  <li>Is the variability homogeneous? why do you think it is like this in terms of viral-host interaction evolution?</li>
  <li>Check in particular the genomic region for 3CLpro. Can you give a rough measure of the percentage of variation of Spike and 3CLpro?</li>
</ol>

<h2 id="3-rational-drug-discovery">3) Rational Drug Discovery</h2>

<h3 id="ligand">Ligand</h3>

<ol>
  <li>Go to the <a href="https://go.drugbank.com/">DRUGBANK</a> web site, and check for the Nirmatrelvir file. Download the structure in the PDB format and visualize it in Chimera. Paste it here Does it look right to you? What is missing?</li>
</ol>

<table>
  <thead>
    <tr>
      <th style="text-align: center"><img src="../figures/nirmatrelvir.png" alt="" /></th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: center">Some related molecules with antiviral properties, including Nirmatrelvir <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9425786/">(Joyce et al., 2022)</a></td>
    </tr>
  </tbody>
</table>

<ol>
  <li>Try fixing the structure using <a href="http://www.cheminfo.org/Chemistry/Cheminformatics/FormatConverter/index.html">openbabel</a>. Paste the new structure as seen in Chimera.</li>
  <li>Can you identify the different functional groups. Discover the protein target of this molecule. Which of them is relevant for the interaction with the target?</li>
</ol>

<h3 id="complex">Complex</h3>

<ol>
  <li>Can you find a structure in the PDB database that contains Nirmatrelivir with its target?</li>
  <li>The target is based on a conserved catalytic dyad, Can you recognize it using Chimera?</li>
  <li>Check the variability of the target and show in the structure where those variants at the level of aminoacids appear. Are they relevant for the function?</li>
  <li>What is the mode of interaction between ligand and target? Can you elaborate on why would you consider it strong and specific? How can this be related to the activity of the protein?</li>
  <li>Can you find information about the way Nirmatrelvir was designed? In particular, what are its precursors?</li>
</ol>

<h1 id="references">References</h1>

<ul>
  <li>You can find interesting material on COVID19 structural biology at the web of the PDB: <a href="https://www.rcsb.org/news/feature/5e74d55d2d410731e9944f52">COVID-19/SARS-CoV-2 Resources</a>.</li>
  <li>Greasley, S. E.; Noell, S.; Plotnikova, O.; Ferre, R.; Liu, W.; Bolanos, B.; Fennell, K.; Nicki, J.; Craig, T.; Zhu, Y.; Stewart, A. E.; Steppan, C. M. <a href="https://doi.org/10.1016/j.jbc.2022.101972">Structural Basis for the in Vitro Efficacy of Nirmatrelvir against SARS-CoV-2 Variants</a>. Journal of Biological Chemistry 2022, 298 (6), 101972.</li>
  <li>Ryan P. Joyce, Vivian W. Hu &amp; Jun Wang <a href="https://link.springer.com/article/10.1007/s00044-022-02951-6">The history, mechanism, and perspectives of nirmatrelvir (PF-07321332): an orally bioavailable main protease inhibitor used in combination with ritonavir to reduce COVID-19-related hospitalizations</a>. Medicinal Chemistry Research 2022; 31, 1637-1646.</li>
</ul>

<p>© <a href="https://mon.uvic.cat/cbbl/members/">Jordi Villà Freixa</a> 2023</p>
